salt/modules/nilrt_ip.py

```diff
@@ -351,39 +351,46 @@ def set_dhcp_linklocal_all(interface):
     ipv4['Gateway'] = dbus.String('', variant_level=1)
     try:
         service.set_property('IPv4.Configuration', ipv4)
-        service.set_property('Domains.Configuration', [''])  # reset domains list
+        service.set_property('Nameservers.Configuration', [''])  # reset nameservers list
     except Exception as exc:
         raise salt.exceptions.CommandExecutionError('Couldn\'t set dhcp linklocal for service: {0}\nError: {1}\n'.format(service, exc))
     return True
 
 
-def set_static_all(interface, address, netmask, gateway, domains):
+def set_static_all(interface, address, netmask, gateway, nameservers):
     '''
     Configure specified adapter to use ipv4 manual settings
 
     :param str interface: interface label
     :param str address: ipv4 address
     :param str netmask: ipv4 netmask
     :param str gateway: ipv4 gateway
-    :param str domains: list of domains servers separated by spaces
+    :param str nameservers: list of nameservers servers separated by spaces
     :return: True if the settings were applied, otherwise an exception will be thrown.
     :rtype: bool
 
     CLI Example:
 
     .. code-block:: bash
 
-        salt '*' ip.set_static_all interface-label address netmask gateway domains
+        salt '*' ip.set_static_all interface-label address netmask gateway nameservers
     '''
     service = _interface_to_service(interface)
     if not service:
         raise salt.exceptions.CommandExecutionError('Invalid interface name: {0}'.format(interface))
     validate, msg = _validate_ipv4([address, netmask, gateway])
     if not validate:
         raise salt.exceptions.CommandExecutionError(msg)
-    validate, msg = _space_delimited_list(domains)
-    if not validate:
-        raise salt.exceptions.CommandExecutionError(msg)
+    if nameservers is not '':
```

Comment:
Perhaps just (to be more pythonic):

```
if nameservers:
```

Reply:
I just want to be sure. I will change it. Same below